components/atom/badge/src/index.scss

```diff
@@ -50,22 +66,21 @@ $tags: success $c-success, error $c-error, alert $c-alert, info $c-gray;
     vertical-align: middle;
   }
 
-  @each $tags-type in $tags {
-    $type: nth($tags-type, 1);
-    $color: nth($tags-type, 2);
+  @each $type, $element in $badges {
+    $bgc: map-get($element, bgc);
+    $c: map-get($element, c);
 
     &-#{$type} {
-      background-color: $color;
-      color: $c-contrast-atom-badge;
-      fill: $c-contrast-atom-badge;
+      background-color: $bgc;
+      color: $c;
+      fill: $c;
     }
 
     &-#{$type}--transparent {
       background-color: transparent;
-      color: $color;
-      fill: $color;
+      color: $bgc;
+      fill: $bgc;
+      padding: 0;
```

Comment:
can you tell us why you need padding 0 ?

Reply:
@davidbarna When it is not framed with a background, padding visually adds space. If you look at the dropbox paper there is no defined padding for transparent badges.

Reply:
![image](https://user-images.githubusercontent.com/16401219/42946959-8157f400-8b6c-11e8-9557-0e7cfd61b27e.png)